File: help/pt_pt/exercicios_da_ficha_5/Ficha05_010/Ficha05_010.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    var minimo = 999 ;
    for( var i = 1; i < 6; i = i + 1 ) {
        document.write(i + "º nota : " );// print text
        nota = parseInt(prompt(i + "º nota : " ,"0") );
        
        document.write(nota + "\n");// print input 
        if(nota < minimo ){
            minimo = nota ;
        }
    }
    document.write("Nota mínima = " + minimo );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 16:04

File: help/pt_pt/exercicios_da_ficha_5/Ficha05_010/Ficha05_010.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha05_010 {

    public:
        int main() {
            int minimo  = 999 ;
            int i  = 0 ;
            for( int i = 1; i < 6; i = i + 1 ) {
                cout << i  << "º nota : ";
                int nota;
                cin >> nota;
                if(nota < minimo ){
                    minimo = nota ;
                }
            }
            cout << "Nota mínima = " << minimo ;
            return 0;
        }
};// end of class Ficha05_010
// Execute main function
int main(){
    Ficha05_010 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 16:04
